### ISSUE TITLE
Problem with case sensitivity

### ISSUE THREAD

**Issue by werunom** [RambIT-nl/highlight-counter#16]
1. In the extension settings, disable case sensitivity
2. In a file, when a word is selected, the extension's "Highlighted" counter in the status bar, shows the right number of case insensitive occurrences 
3. But, in the editor, only case-sensitive instances are highlighted. 

<img width="231" alt="Screenshot 2023-02-12 at 12 25 49" src="https://user-images.githubusercontent.com/24664855/218297496-8058a871-6b43-4a14-8e25-235b8f1210e3.png">

VS Code Version: 1.75.1
OS Version: Mac OS Ventura 13.2

Please note that, there is a related issue about case-sensitive highlighting in the current release. See [here](https://github.com/microsoft/vscode/issues/174173).

**Comment by ramonbakker** [comment]
Thanks for reporting.

The extension counts occurrences of the highlighted text based on the text in a document, not based on highlighted parts. It seems like VS Code made the highlighting case sensitive. In the example in issue #4, the highlighting is case insensitive.

If VS Code adds an option to toggle case sensitivity for the highlight functionality, we can count occurrences based on that setting.

**Comment by werunom** [comment]
So, looks like this is not the extension's bug; but of vscode. In any case, lets see what the dev team does with the bug reported there.

**Comment by werunom** [comment]
Looks like the above bug is no more reproducible. See [this](https://github.com/microsoft/vscode/issues/174173#issuecomment-1427785041). Even though the screenshot was taken with extensions disabled. I could see the same behaviour when the extensions were enabled again. 

have no clue how this was not the case when I raised the above bug. 🤷🏼‍♂️ 

Excuse for the false alarm. Closing this. (but the original bug for vscode still remains to be solved.)
